tools/packaging/kata-deploy/runtimeclasses/kata-qemu-nvidia-gpu-snp.yaml

```diff
@@ -0,0 +1,13 @@
+---
+kind: RuntimeClass
+apiVersion: node.k8s.io/v1
+metadata:
+    name: kata-qemu-nvidia-gpu-snp
+handler: kata-qemu-nvidia-gpu-snp
+overhead:
+    podFixed:
+        memory: "160Mi"
```

Comment:
I am seeing the values for `cpu` and `memory` for `kata-qemu-snp` are `1.0` and `2048Mi`, respectively. I would be grateful if you could explain a rationale for these reduced estimation. Thanks!

Reply:
Missed that, those were the "old" default values, let me update.

Reply:
Done!